src/BinlogTool/DumpRecords.cs

```diff
@@ -0,0 +1,100 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.Build.Logging.StructuredLogger;
+using Microsoft.Build.Logging;
+using StructuredLogger.Utils;
+
+namespace BinlogTool
+{
+    internal static class DumpRecords
+    {
+        public static void Run(List<string> inputs, bool includeTotal, bool includeRollup, bool includeDetails)

```

Comment:
not sure we need to suppose multiple binlogs here, but I don't care either way is fine

Reply:
we can leave as is

Reply:
I got this for free from redactor (where it's helpful)... but true it might not have usage here.
We can remove later (or better extract so the code is on single place)